tracker/dmlc_tracker/ssh.py

```diff
@@ -20,23 +21,68 @@ def sync_dir(local_dir, slave_node, slave_dir):
         slave_node[1], local_dir, remote)
     subprocess.check_call([prog], shell = True)
 
-def get_env(pass_envs):
+def prepare_envs(args):
+    """
+    Load environment variables from arguments
+    """
+    envs = {}
+    # default env variables which are always passed by the system
+    envs['default'] = {'OMP_NUM_THREADS', 'KMP_AFFINITY', 'LD_LIBRARY_PATH', 'AWS_ACCESS_KEY_ID',
```

Comment:
I'm not sure if it makes sense to have these defaults

Reply:
These were passed earlier, so I kept them as is. If the variable is not set locally(on the machine which launch.py is running), then it has no effect anyway. LD_LIBRARY_PATH is one that seems a bit strange considering different machines might not have similar installations. But for most cases where uses are running on similar setups, it should be fine.

Reply:
Which ones do you think we should remove?